src/js/components/Box/__tests__/Box-test.tsx

```diff
@@ -735,6 +735,22 @@ describe('Box', () => {
     expect(container.firstChild).toMatchSnapshot();
   });
 
+  test('renders border=between and gap=pixle value', () => {
+    const { container } = render(
+      <Grommet>
+        <Box gap="12px" border="top" pad="medium">
```

Comment:
Should this be 'between' instead of 'top'
```suggestion
        <Box gap="12px" border="between" pad="medium">
```

Reply:
opps yes thank you